patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -183,22 +196,25 @@
                      {
                          d1 = d11;
                          d13 = this.field_70163_u + 2.0D;
-@@ -1700,6 +1724,7 @@
+@@ -1700,6 +1726,7 @@
          }
  
          this.field_70160_al = true;
 +        net.minecraftforge.common.ForgeHooks.onLivingJump(this);
      }
  
      protected void func_70629_bd()
-@@ -1972,6 +1997,7 @@
+@@ -1972,6 +1999,7 @@
  
      public void func_70071_h_()
      {
 +        if (net.minecraftforge.common.ForgeHooks.onLivingUpdate(this)) return;
          super.func_70071_h_();
          this.func_184608_ct();
  
+<<<<<<< HEAD
+@@ -2488,6 +2516,40 @@
+=======
```

Comment:
sorry I forgot to unassign you when they had to resolve new conflicts. already asked them to fix this.